rust/chains/tw_solana/src/transaction/versioned.rs

```diff
@@ -110,6 +114,74 @@ impl VersionedMessage {
             VersionedMessage::V0(v0) => v0.recent_blockhash = recent_blockhash,
         }
     }
+
+    pub fn set_priority_fee(
+        &mut self,
+        priority_fee_limit: Option<UnitLimit>,
+        priority_fee_price: Option<UnitPrice>,
+    ) {
+        // If priority fee limit and price are not provided, return early.
+        if priority_fee_limit.is_none() && priority_fee_price.is_none() {
+            return;
+        }
+
+        let (account_keys, header, instructions) = match self {
+            VersionedMessage::Legacy(legacy) => (
+                &mut legacy.account_keys,
+                &mut legacy.header,
+                &mut legacy.instructions,
+            ),
+            VersionedMessage::V0(v0) => {
+                (&mut v0.account_keys, &mut v0.header, &mut v0.instructions)
+            },
+        };
+
+        // Add compute budget account to the account keys if it doesn't exist.
+        if !account_keys.contains(&COMPUTE_BUDGET_ADDRESS) {
+            account_keys.push(*COMPUTE_BUDGET_ADDRESS);
+            header.num_readonly_unsigned_accounts += 1;
```

Comment:
We should also take `address_lookup_table` into account. The lookup indices should be updated as well